### PR TITLE
docs: add base64 example

### PR DESCRIPTION
**Pull request by kantorcodes**
**Description**:
Adds a small tool to test executing a base64 transaction directly. This has been super helpful in testing the SaucerSwap integration. 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: demos/react-dapp/src/App.tsx [hunk 1]
```diff
@@ -589,8 +619,39 @@ const App: React.FC = () => {
             </button>
           </div>
         </section>
-        <hr />
-        <h2>Pairing and session management:</h2>
```
Comment (justynspooner) [inline demos/react-dapp/src/App.tsx:593]: I assume you want these removing as the button title for disconnecting the sessions is obvious enough?

**Review by justynspooner — APPROVED** [review]

File: demos/react-dapp/src/App.tsx (hunk 1 above)
Comment (kantorcodes) [reply demos/react-dapp/src/App.tsx]: > I assume you want these removing as the button title for disconnecting the sessions is obvious enough?

haha no, I removed this by accident. Thanks for flagging

File: demos/react-dapp/src/App.tsx [hunk 2]
```diff
@@ -273,25 +276,58 @@ const App: React.FC = () => {
...
+      setIsLoading(false)
+    }
+  }
   // Create multi-signature account
   const handleCreateMultisigAccount = async () => {
     // Fetch public keys from mirror node for each account
     const fetchPublicKey = async (accountId: string) => {
       const response = await fetch(
-        `https://testnet.mirrornode.hedera.com/api/v1/accounts/${accountId}`
+        `https://testnet.mirrornode.hedera.com/api/v1/accounts/${accountId}`,
```
Comment (tmctl) [inline demos/react-dapp/src/App.tsx:312]: don't need a comma here, is this part of the prettier config?

Comment (kantorcodes) [reply demos/react-dapp/src/App.tsx:312]: Yup, that's because of prettier. It's also best practice to add trailing commas.

**Review by tmctl — APPROVED** [review]
lgtm

**Pull request merged**